app/views/pages/_campaigner_overlay.slim

```diff
@@ -17,9 +17,20 @@
             = f.hidden_field :publish_status, class: 'campaigner-overlay__publish-field', value: page.published? ? "unpublished" : "published"
             = f.submit "Publish", class: 'campaigner-overlay__toggle-publish campaigner-overlay__info--inactive'
             = f.submit "Unpublish", class: 'campaigner-overlay__toggle-publish campaigner-overlay__info--active', 'data-confirm' => "Are you sure you want to unpublish this page? It will become inaccessible except to logged-in campaigners."
+        .campaigner-overlay__copy-url-link
```

Comment:
```suggestion
       .campaigner-overlay__copy-url-link
          | Copy URL
        span.copied style="display: none;"
          = "Copied!"
```

We need a visual indication that the link was copied to the clipboard, let's add it, see the example above with the hidden span